TESTING.md

```diff
@@ -79,8 +63,10 @@ Also run the config render script. If you are planning on running the firecloud
 	* Defaults to `$PWD`.
 * Vault auth token
 	* Defaults to reading it from the .vault-token via `$(cat ~/.vault-token)`.
-* `dev`
-	* Environment of your FiaB.
+* env
+	* Environment of your FiaB; defaults to `dev`
+* branch of firecloud-automated-testing
+    * Configs branch; defaults to `master`
```

Comment:
Assuming this is the doc for the positional argument before "env", it should be moved above the doc for "env". Being positional arguments, the docs for them should be in the same order.